pass.el

```diff
@@ -342,6 +345,54 @@ indented according to INDENT-LEVEL."
           (forward-line -1)
           (pass-closest-entry)))))
 
+(defun pass-otp-options (option)
+  "Dispatch otp actions depending on user OPTION input.
+Display help message with OTP functionality options."
+  (interactive
+   (list
+    (read-char-choice "[i] Insert, [a] Append, [s] Append from screenshot, [o] Copy token, [u] Copy URI, or [C-g] to abort: "
+                      '(?i ?a ?s ?o ?u))))
+  (unless (require 'password-store-otp nil t)
+    (user-error "You cannot do this without installing `password-store-otp' first"))
+  (pcase option
+    (?i (pass-otp-insert))
+    (?a (pass-otp-append))
+    (?s (pass-otp-from-screenshot))
+    (?o (pass-otp-token-copy))
+    (?u (pass-otp-uri-copy))))
+
+(defun pass-otp-token-copy ()
+  "Add OTP Token from closest entry to kill ring."
+  (interactive)
+  (pass--with-closest-entry entry
+    (password-store-otp-token-copy entry)))
+
+(defun pass-otp-uri-copy ()
+  "Add OTP URI from closest entry to kill ring."
+  (interactive)
+  (pass--with-closest-entry entry
+    (password-store-otp-uri-copy entry)))
+
+(defun pass-otp-insert ()
+  "Insert an OTP URI entry to the password-store.
+The password is read from user input."
+  (interactive)
+  (call-interactively #'password-store-otp-insert)
+  (pass-update-buffer))
+
+(defun pass-otp-append ()
+  "Append an OTP URI to an existing entry in the password-store.
+The password is read from user input."
+  (interactive)
+  (call-interactively #'password-store-otp-append)
+  (pass-update-buffer))
+
+(defun pass-otp-from-screenshot ()
+  "Append an OTP URI taken from a screenshot to an existing entry in the password-store."
+  (interactive)
+  (call-interactively #'password-store-otp-append-from-image)
```

Comment:
same here

Reply:
This one I can easily fix because it's corresponding `password-store-otp` function only needs the `entry` argument. I didn't notice that before. Thanks!